charts/datafold/charts/clickhouse/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: clickhouse
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "22.8.4.7-df5"
```

Comment:
At some point this should probably become a setting elsewhere. Not sure how to deal with it yet